lib/ups/builders/builder_base.rb

```diff
@@ -218,8 +218,8 @@ def shipment_service_options
         end
       end
 
-      def packaging_type
-        code_description 'PackagingType', '02', 'Customer Supplied'
+      def packaging_type(packaging_type)
+        code_description 'PackagingType', packaging_type[:code], packaging_type[:description]
```

Comment:
This is a breaking change as such would require a major bump. I would offer to gracefully fallback to previous options if not present